bluebutton-data-model-rif-samples/src/main/java/gov/hhs/cms/bluebutton/data/model/rif/samples/StaticRifResourceGroup.java

```diff
@@ -22,7 +22,9 @@
 
 	SAMPLE_C(StaticRifResource.SAMPLE_C_BENES, StaticRifResource.SAMPLE_C_CARRIER, StaticRifResource.SAMPLE_C_INPATIENT,
 			StaticRifResource.SAMPLE_C_OUTPATIENT, StaticRifResource.SAMPLE_C_SNF, StaticRifResource.SAMPLE_C_HOSPICE,
-			StaticRifResource.SAMPLE_C_HHA, StaticRifResource.SAMPLE_C_DME, StaticRifResource.SAMPLE_C_PDE);
+			StaticRifResource.SAMPLE_C_HHA, StaticRifResource.SAMPLE_C_DME, StaticRifResource.SAMPLE_C_PDE),
+
+	SAMPLE_U(StaticRifResource.SAMPLE_U_BENES, StaticRifResource.SAMPLE_U_CARRIER);
```

Comment:
You don't have any ITs that use this new sample delta data, best as I can tell. Can you add a new test case that verifies they work as expected?

Reply:
Gosh darned it. I have -- yet again -- gotten confused by the split between `bluebutton-data-model` and `bluebutton-data-pipeline`. My apologies: I see `RifLoaderIT.updateSampleU()` now. Sigh.

Reply:
no worries...I created a couple more tests to ensure the updates were working correctly.